src/cluster/cluster.cc

```diff
@@ -567,10 +565,10 @@ std::string Cluster::genNodesDescription() {
   return nodes_desc;
 }
 
-std::map<std::string, std::string> Cluster::getClusterNodeSlots() const {
+std::map<std::string, std::string, std::less<>> Cluster::getClusterNodeSlots() const {
```

Comment:
I don't think we should specify the compare type explicitly.

If it reported by sonarcloud, please suppress it.

Reply:
Yes, It is according to https://sonarcloud.io/organizations/apache/rules?open=cpp%3AS6045&rule_key=cpp%3AS6045.
We can disable it in [Quality Profiles](https://sonarcloud.io/project/quality_profiles?id=apache_kvrocks)

Reply:
Maybe this is for find the string_view?

Reply:
Hmm I got the motivation. Sorry we can keep them.